lms/djangoapps/grades/config/waffle.py

```diff
@@ -2,7 +2,7 @@
 This module contains various configuration settings via
 waffle switches for the Grades app.
 """
-from openedx.core.djangolib.waffle_utils import WaffleSwitchPlus
+from openedx.core.djangoapps.waffle_utils import WaffleSwitchPlus
```

Comment:
How about calling the Django app ``openedx.core.djangoapps.feature_flags`` and then it can encompass Waffle features, configuration models, and might even eventually use a different technology than Waffle.  I could even picture us making this into its own repo (open-edx-feature-flags) so that all IDAs could benefit from using Waffle flags that are context aware.